api/src/main/resources/db/migration/V1.0.20__EDX_API.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE SECURE_EXCHANGE_STUDENT
+(
+    SECURE_EXCHANGE_STUDENT_ID UUID                 NOT NULL,
+    STUDENT_ID                 UUID                 NOT NULL,
+    SECURE_EXCHANGE_ID         UUID                 NOT NULL,
+    CREATE_USER                VARCHAR(32)          NOT NULL,
+    CREATE_DATE                TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    CONSTRAINT SECURE_EXCHANGE_STUDENT_ID_PK PRIMARY KEY (SECURE_EXCHANGE_STUDENT_ID)
+);
+
+ALTER TABLE SECURE_EXCHANGE_STUDENT
+    ADD CONSTRAINT FK_SECURE_EXCHANGE_STUDENT_SECURE_EXCHANGE_ID FOREIGN KEY (SECURE_EXCHANGE_ID) REFERENCES SECURE_EXCHANGE (SECURE_EXCHANGE_ID);
+
+CREATE INDEX SECURE_EXCHANGE_STUDENT_SECURE_EXCHANGE_ID_I ON SECURE_EXCHANGE_STUDENT (SECURE_EXCHANGE_ID);
```

Comment:
@cditcher - Did you shorten the index name due to character length?

Reply:
No I was just following the other index names. Should it be different?